### PR TITLE
EVP Automated Test LPS-186802|master

### PR DESCRIPTION
**Pull request by juliosilveira77**
https://liferay.atlassian.net/browse/LPS-186802

### CODE REVIEW

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

**Comment by juliosilveira77** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 1]
```diff
@@ -22,18 +22,16 @@ definition {
...
-		ObjectAdmin.deleteAllCustomObjectsViaAPI();
-
-		if (${testLiferayVirtualInstance} == "true") {
-			PortalInstances.tearDownCP();
-		}
-		else {
-			JSONGroup.deleteGroupByName(groupName = "EVP Portal");
-		}
-	}
+	//tearDown {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:25]: Hey @juliosilveira77 , this code should be a comment?

case should please remove this block. lets avoid push comment

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 2]
```diff
@@ -107,4 +105,82 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-180358. Verify all status changes after 'need more info' request"
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:108]: @juliosilveira77 I`m trying a new approach for test description:
@description = "LPS-180358 - Organization status should changes after 'need more info' request' flow"

can you validate with gabi?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 3]
```diff
@@ -107,4 +105,82 @@ definition {
...
+	@priority = 5
+	test VerifyStatusAfterRequestMoreInfo {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:110]: please validate with Heidi her new macro to perform this organization creation flow

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 4]
```diff
@@ -107,4 +105,82 @@ definition {
...
+				organizationName = "Organization 5",
+				organizationSite = "Social Media Link",
+				smallDescription = "Small Description",
+				state = "State",
+				street = "Street",
+				taxId = "Tax Id",
+				zipCode = "Zip");
+		}
+
+		task ("When the Manager click in 'Need more info'") {
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:139]: this step action should be  When the Manager requests more info.

When the Manager click in 'Need more info shows only one action.

**Review by renato-araujo-liferay — CHANGES_REQUESTED** [review]
Hey @juliosilveira77 can you check my final changes request?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase (hunk 4 above)
Comment (juliosilveira77) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:139]: The flow of my test is different from the macro it is creating, I took this doubt with it.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase (hunk 3 above)
Comment (juliosilveira77) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:110]: The flow of my test is different from the macro it is creating, I took this doubt with it.

**Comment by juliosilveira77** [comment]
ci:test:sf

**Review by renato-araujo-liferay — APPROVED** [review]

**Comment by renato-araujo-liferay** [comment]
ci:forward

**Pull request closed without merging**